jekyll/index.markdown

```diff
@@ -571,7 +571,7 @@ default gems, except for
 - Gems that only appear under the `:development` group
 - All Ruby files under `test/**/*.rb`
 
-This behavior can be overridden and tuned. Learn how to configure it [for VS Code](https://github.com/Shopify/ruby-lsp/tree/main/vscode#indexing-configuration).
+This behavior can be overridden and tuned. Learn how to configure it [for VS Code](https://shopify.github.io/ruby-lsp/vscode-extension.html#indexing-configuration).
```

Comment:
Let's use relative references in all of these. Otherwise, when you're working on the docs locally, clicking on these links will open the prod version instead of your local.

```suggestion
This behavior can be overridden and tuned. Learn how to configure it [for VS Code](vscode-extension#indexing-configuration).
```

Reply:
Sure!